templates/docs/whats-new.md

```diff
@@ -27,6 +27,12 @@ When we add, make significant updates, or deprecate a component we update their
       <td>2.36.0</td>
       <td>We've added a variation of the accordion using tick elements</td>
     </tr>
+    <tr>
```

Comment:
No need for this, we are not adding new class name or anything. Just fixing a bug with a color.